lib/internal/bootstrap/node.js

```diff
@@ -372,3 +372,26 @@ function defineOperation(target, name, method) {
     value: method
   });
 }
+
+function setUpCodegenFromStringCallback(bindings) {
+
+  const codeGenerationFromStringsAllowed = bindings
+      .codeGenerationFromStringsAllowed();
+
+  if (!codeGenerationFromStringsAllowed) {
+    return; // We do nothing if it is not allowed to perform code
```

Comment:
The comments here are kind of awkward. Maybe shorten the text to something like:

```js
// Do nothing if code generation from strings is not allowed.
```

And move it right above the `return` statement to avoid breaking the comment across lines.